File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/Unfollow.java
package me.grishka.houseclub.api.methods;

import me.grishka.houseclub.api.BaseResponse;
import me.grishka.houseclub.api.ClubhouseAPIRequest;

public class Unfollow extends ClubhouseAPIRequest<BaseResponse> {
    public Unfollow(int userID) {
        super("POST", "unfollow", BaseResponse.class);
        requestBody = new Body(userID);
    }

    private static class Body {
        public int userId;

        public Body(int userId) {
            this.userId = userId;
        }
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/Unfollow.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.BaseResponse
import me.grishka.houseclub.api.ClubhouseAPIRequest

class Unfollow(userID: Int) : ClubhouseAPIRequest<BaseResponse?>("POST", "unfollow", BaseResponse::class.java) {
    private class Body(var userId: Int)

    init {
        requestBody = Body(userID)
    }
}